can_tools/scrapers/official/federal/CDC/cdc_coviddatatracker.py

```diff
@@ -1,108 +0,0 @@
-import pathlib
-from typing import Dict
-
-import pandas as pd
-import requests
-import us
-import multiprocessing
-
-from can_tools.scrapers.base import CMU
-from can_tools.scrapers.official.base import FederalDashboard
-
-
-class CDCCovidDataTracker(FederalDashboard):
-    has_location = True
-    location_type = "county"
-    source = "https://covid.cdc.gov/covid-data-tracker/#county-view"
-    source_name = "Centers for Disease Control and Prevention"
-    provider = "cdc"
-
-    variables = {
-        "new_cases_7_day_rolling_average": CMU(
-            category="cases", measurement="rolling_average_7_day", unit="people"
-        ),
-        "new_deaths_7_day_rolling_average": CMU(
-            category="deaths", measurement="rolling_average_7_day", unit="people"
-        ),
-        "percent_new_test_results_reported_positive_7_day_rolling_average": CMU(
-            category="pcr_tests_positive",
-            measurement="rolling_average_7_day",
-            unit="percentage",
-        ),
-        "new_test_results_reported_7_day_rolling_average": CMU(
-            category="pcr_tests_total",
```

Comment:
I just sent email to the CMU folks, but I think they are relying on this field.  So we may need to keep this scraper. 🙁